sormas-backend/src/main/java/de/symeda/sormas/backend/importexport/ImportFacadeEjb.java

```diff
@@ -221,10 +221,16 @@ public void generateCaseImportTemplateFile() throws IOException {
 		appendListOfFields(importColumns, CaseDataDto.class, "", separator);
 		appendListOfFields(importColumns, SampleDto.class, "", separator);
 		appendListOfFields(importColumns, PathogenTestDto.class, "", separator);
+		addPrimaryPhoneAndEmail(separator, importColumns, PersonDto.PHONE, PersonDto.EMAIL_ADDRESS);
 
 		writeTemplate(Paths.get(getCaseImportTemplateFilePath()), importColumns, true);
 	}
 
+	private void addPrimaryPhoneAndEmail(char separator, List<ImportColumn> importColumns, String phone, String emailAddress) {
```

Comment:
Since the method specifies that it adds the phone and email column why is it needed to also specify the `phone` and `emailAddress` strings?